workflow/executor/resource.go

```diff
@@ -372,9 +372,13 @@ func runKubectl(args ...string) ([]byte, error) {
 	}()
 	var buf bytes.Buffer
 	if err := kubectlcmd.NewKubectlCommand(kubectlcmd.KubectlOptions{
-		Arguments:   args,
-		ConfigFlags: genericclioptions.NewConfigFlags(true),
-		IOStreams:   genericclioptions.IOStreams{Out: &buf, ErrOut: os.Stderr},
+		Arguments: args,
+		// TODO(vadasambar): re-visit this once https://github.com/kubernetes/kubernetes/pull/120024 is merged
```

Comment:
Can you add add instructions on how to update this part once that PR is merged? (in case in you are away we can also remove the TODO in the future)

Reply:
Thank you for the comment. I have re-worded the TODO explaining what needs to be done. Let me know if you think we should reword it (or add more).